### DISCUSSION TITLE
Link + select

### DISCUSSION THREAD

**Discussion by rodbs** [remix-run/remix#4406] [Q&A] [answered]
Hi,

Is it possible to use a `Link` within a `select`? I'd like to link to a nested route based on the dropdown selection.

thkx!!

**Comment by sergiodxa** [comment, marked as answer]
You should listen for the change event of the select and use the navigate function there

**Reply by rodbs** [reply]
I understand you mean the `navigate` from `useNavigate`, right?
https://reactrouter.com/en/6.4.1/hooks/use-navigate

Because the `redirect` function only works on the server?

**Reply by sergiodxa** [reply]
Yes, the navigate function returned by the useNavigate hook.

**Reply by rodbs** [reply]
thx!!
